pontoon/batch/utils.py

```diff
@@ -114,11 +119,32 @@ def find_and_replace(translations, find, replace, user):
         translation.rejected_date = None
         translation.rejected_user = None
         translation.fuzzy = False
-        translations_to_create.append(translation)
+
+        errors = run_checks(
+            translation.entity,
+            translation.locale.code,
+            translation.entity.string,
+            translation.string,
+            use_tt_checks=False,
+        )
```

Comment:
I didn't check, but I wonder in how many DB queries will this result. Looking at the surrounding code, we might need to prefetch `entity__resource` and `locale`.